sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/windowing/PaneInfo.java

```diff
@@ -411,5 +413,20 @@ public PaneInfo decode(final InputStream inStream) throws CoderException, IOExce
 
     @Override
     public void verifyDeterministic() {}
+
+    @Override
+    protected long getEncodedElementByteSize(PaneInfo value) throws Exception {
```

Comment:
can you add some unit tests creating various elements and comparing this and actual serialized size?